sklearn/model_selection/tests/test_split.py

```diff
@@ -633,16 +633,17 @@ def test_stratified_shuffle_split_init():
     assert_raises(ValueError, next,
                   StratifiedShuffleSplit(3, 0.2).split(X, y))
 
+    X = np.arange(9)
+    y = np.asarray([0, 0, 0, 1, 1, 1, 2, 2, 2])
+
     # Check that error is raised if the test set size is smaller than n_classes
```

Comment:
@jnothman the test written below wasn't testing what comment said.

Reply:
of course I might be wrong in my assumption, I am not sure anymore!

Reply:
To avoid confusion in the future, we can use

```py
msg = "The test_size = 2 should be greater or equal to the number of classes = 3"
with pytest.raises(ValueError, match=msg):
    next(StratifiedShuffleSplit(3, 2).split(X, y))
```

so we can check for the error message explicitly.

Reply:
will add these to the messages!